packages/web/hooks/use-feature-flags.ts

```diff
@@ -51,22 +55,39 @@ const defaultFlags: Record<ModifiedFlags, boolean> = {
   positionRoi: true,
   swapToolSimulateFee: false,
   portfolioPageAndNewAssetsPage: false,
-  newAssetsPage: false,
+  newAssetsPage: true,
   displayDailyEarn: false,
   newDepositWithdrawFlow: false,
   oneClickTrading: false,
+  limitOrders: true,
   advancedChart: false,
   _isInitialized: false,
   _isClientIDPresent: false,
 };
 
+const LIMIT_ORDER_COUNTRY_CODES =
+  process.env.NEXT_PUBLIC_LIMIT_ORDER_COUNTRY_CODES?.split(",").map((s) =>
+    s.trim()
+  ) ?? [];
+
 export const useFeatureFlags = () => {
   const launchdarklyFlags: Record<AvailableFlags, boolean> = useFlags();
   const { isMobile } = useWindowSize();
   const [isInitialized, setIsInitialized] = useState(false);
 
   const client = useLDClient();
 
+  const { data: levanaGeoblock } = useQuery(
+    ["levana-geoblocked"],
+    () =>
+      apiClient<LevanaGeoBlockedResponse>("https://geoblocked.levana.finance/"),
+    {
+      staleTime: Infinity,
+      cacheTime: Infinity,
```

Comment:
Could this cause issues if the user changes location?

Reply:
Do you want me to adjust this for both use cases? It could be an issue if someone travels during geolocation blocking but that's only going to be in place for a few days.

The worse case is the use in `_app.tsx`.

Reply:
Yeah I think we should not use Infinity. Rather like 1d or something